definitions/ext-host/summary_metrics.yml

```diff
@@ -0,0 +1,36 @@
+localIp:
+  tag:
+    key: localIp
+  title: IP
+  unit: STRING
+
+shortHostname:
+  tag:
+    key: shortHostname
+  title: hostname
+  unit: STRING
+
+cpuPercent:
+  title: CPU Percent
+  unit: PERCENTAGE
+  queries:
+    flexagent:
```

Comment:
same here. a query for all sources should be provided

Reply:
added